packages/angular-cli/models/webpack-build-development.ts

```diff
@@ -1,14 +1,3 @@
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-
 export const getWebpackDevConfigPartial = function(projectRoot: string, appConfig: any) {
-  return {
-    output: {
-      filename: '[name].bundle.js',
-      sourceMapFilename: '[name].bundle.map',
-      chunkFilename: '[id].chunk.js'
-    },
-    plugins: [
-      new ExtractTextPlugin({filename: '[name].bundle.css'})
-    ]
-  };
+  return { };
```

Comment:
Also wonder if that would help with the reload times tbh. It sounds like it might do away with rebuilding some modules due to id changes.